packages/expo-calendar/src/next/ExpoCalendar.types.ts

```diff
@@ -0,0 +1,394 @@
+   * @platform ios
+   */
+  url?: string;
+  /**
+   * Description or notes saved with the event.
+   */
+  notes: string;
+  /**
+   * Array of Alarm objects which control automated reminders to the user.
+   */
+  alarms: Alarm[];
+  /**
+   * Object representing rules for recurring or repeating events. Set to `null` for one-time events.
+   */
+  recurrenceRule: RecurrenceRule | null;
+  /**
+   * Date object or string representing the time when the event starts.
+   */
+  startDate: string | Date;
+  /**
+   * Date object or string representing the time when the event ends.
+   */
+  endDate: string | Date;
+  /**
+   * For recurring events, the start date for the first (original) instance of the event.
+   * @platform ios
+   */
+  originalStartDate?: string | Date;
+  /**
+   * Boolean value indicating whether or not the event is a detached (modified) instance of a recurring event.
+   * @platform ios
+   */
+  isDetached?: boolean;
+  /**
+   * Whether the event is displayed as an all-day event on the calendar
+   */
+  allDay: boolean;
+  /**
+   * The availability setting for the event.
+   */
+  availability: Availability;
+  /**
+   * Status of the event.
+   */
+  status: EventStatus;
+  /**
+   * Organizer of the event.
+   * This property is only available on events associated with calendars that are managed by a service ie. Google Calendar or iCloud.
+   * The organizer is read-only and cannot be set.
+   *
+   * @platform ios
+   */
+  organizer?: Organizer;
+  /**
+   * Email address of the organizer of the event.
+   * @platform android
+   */
+  organizerEmail?: string;
+  /**
+   * User's access level for the event.
+   * @platform android
+   */
+  accessLevel?: EventAccessLevel;
+  /**
+   * Whether invited guests can modify the details of the event.
+   * @platform android
+   */
+  guestsCanModify?: boolean;
+  /**
+   * Whether invited guests can invite other guests.
+   * @platform android
+   */
+  guestsCanInviteOthers?: boolean;
+  /**
+   * Whether invited guests can see other guests.
+   * @platform android
+   */
+  guestsCanSeeGuests?: boolean;
+  /**
+   * For detached (modified) instances of recurring events, the ID of the original recurring event.
+   * @platform android
+   */
+  originalId?: string;
+  /**
+   * For instances of recurring events, volatile ID representing this instance. Not guaranteed to
+   * always refer to the same instance.
+   * @platform android
+   */
+  instanceId?: string;
+
+  /**
+   * Launches the calendar UI provided by the OS to preview an event.
+   * @return A promise which resolves with information about the dialog result.
+   * @header systemProvidedUI
+   */
+  openInCalendarAsync(
+    params: CalendarDialogOpenParamsNext | null // TODO: Support skipping this param instead of passing null, change needed in the core
+  ): void;
+
+  /**
+   * Launches the calendar UI provided by the OS to edit or delete an event. On Android, this is the same as `openEventInCalendarAsync`.
```

Comment:
`openEventInCalendarAsync` doesn't exist anywhere in this module, also this method is marked as iOS only so not too much point documenting Android behavior imo